### DISCUSSION TITLE
My Achievements are not showing in GitHub profile.

### DISCUSSION THREAD

**Discussion by anderscain63** [community/community#56662] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, Gentlemen! Already did everything to get GitHub Achievements on my profile but not showing. Can you explain what I did incorrectly or how I can easily achievement? If you have any suggestions for quick videos or other original ideas, kindly let me know. Thanks!

**Comment by radar-blip** [comment, marked as answer]
You have Quickdraw achievement but it is hidden from your profile.
Check here to un-hide it: https://github.com/anderscain63?tab=achievements 

Here is info on how to get more [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Reply by anderscain63** [reply]
@linuxlawson Thanks for your note! It works for me.

**Reply by radar-blip** [reply]
You're welcome. Happy to help!
